scripts/qa_crowbarsetup.sh

```diff
@@ -1893,6 +1924,16 @@ function onadmin_post_allocate()
 {
     pre_hook $FUNCNAME
 
+    # for testing the Manila generic driver, the nodes with the m-shr service
+    # and the manila-service VM need an IP from the same subnet. So add all
+    # current nodes to the subnet
+    if iscloudver 6plus; then
+        cmachines=$(get_all_nodes)
```

Comment:
well, we don#t want to have the crowbar node being exposed to the manila network.. yes it strictly doesn't matter, but the setup makes less sense without admin node connected to manila.. (think about GRE or the like, it wouldn't work anyway)